pages/styles/profile.css

```diff
@@ -1,4 +1,6 @@
 .actionItems {
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
https://caniuse.com/#search=space-evenly

how about `space-around`

Reply:
That'll work